debian/changelog

```diff
@@ -1,3 +1,9 @@
+pgcron (1.5.1.citus-1) stable; urgency=low
+
+  * Official 1.5.1 release of Pg Cron
```

Comment:
This is a weird way to write an abbreviation for PostgreSQL. How about dropping the name altogether, or capitalizing PG part?

```suggestion
  * Official 1.5.1 release
```
or
```suggestion
  * Official 1.5.1 release of PG Cron
```

Reply:
Fixed with the second alternative